electron/main.js

```diff
@@ -89,6 +89,15 @@ if (process.platform !== 'darwin') {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+const de = process.env.XDG_CURRENT_DESKTOP;
+if (de.includes('Unity')) {
```

Comment:
There might be a better way to do this test. We need to change from Unity:Unity7 to Unity.